manifests/init.pp

```diff
@@ -152,6 +152,8 @@
     notify  => Service['nftables'],
   }
 
+  # firewalld.enable can be mask or false depending upon if firewalld is installed or not
```

Comment:
Would it be better to move this to the `@param` documentation for `$firewalld_enable`?

Reply:
It's there already.

```puppet
# @param firewalld_enable
#   Configures how the firewalld systemd service unit is enabled. It might be
#   useful to set this to false if you're externaly removing firewalld from
#   the system completely.
```

 Adding the link there is a bit ugly and here in the code makes it obvious as something to review for the future.